tasks/util/browserify_wrapper.js

```diff
@@ -38,7 +38,7 @@ module.exports = function _bundle(pathToIndex, pathToBundle, opts) {
     var browserifyOpts = {};
     browserifyOpts.standalone = opts.standalone;
     browserifyOpts.debug = opts.debug;
-    browserifyOpts.transform = outputMinified ? [compressAttributes] : [];
+    browserifyOpts.transform = outputMinified ? ['glslify', compressAttributes] : ['glslify'];
```

Comment:
Hmm. I think we don't need this patch anymore as the package.json now has `browseirify.transform` field.